### PR TITLE
new file:   Python/other/Minimize_the_Heights.py

### PR DESCRIPTION
**Pull request by soumikbaithalu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes I Have read 

### Description

let's sort arrays then average minimum difference and comparing the minimum difference and print the output 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This is an issue related to #5053

### CODE REVIEW

File: Python/other/Minimize_the_Heights.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+def minimize_the_heights(arr,k):
```
Comment (HarshCasper) [inline Python/other/Minimize_the_Heights.py:7]: Add a beginner friendly description

File: Python/other/Minimize_the_Heights.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Enter K: 3
+
+OUTPUT:
+11
+
+Explanation:
+The array can be modified as {6, 12, 9, 13, 17}. 
+The difference between the largest and the smallest is 17-6 = 11.
+
+'''
```
Comment (HarshCasper) [inline Python/other/Minimize_the_Heights.py]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/other/Minimize_the_Heights.py (hunk 1 above)
Comment (soumikbaithalu) [reply Python/other/Minimize_the_Heights.py:7]: @HarshCasper please check the as you told me to change

Comment (soumikbaithalu) [reply Python/other/Minimize_the_Heights.py:7]: Done @HarshCasper

File: Python/other/Minimize_the_Heights.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#Given an array arr[] denoting heights of len(arr) towers and a positive integer K, 
+#you have to modify the height of each tower either by increasing or decreasing them by K only once.
+#After modifying, height should be a non-negative integer. 
+#Find out what could be the possible minimum difference of the height of shortest and 
+#longest towers after you have modified each tower.
```
Comment (HarshCasper) [inline Python/other/Minimize_the_Heights.py:5]: Use multi-line string here

**Comment by siddharth25pandey** [comment]
@soumik2012 Plagiarized from [here](https://stackoverflow.com/questions/63000076/minimize-the-maximum-difference-between-the-heights)

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Pull request closed without merging**